lib_tflite_micro/src/tflite-xcore-kernels/xcore_load_from_flash_v2.cc

```diff
@@ -52,8 +52,8 @@ void* Init(TfLiteContext* context, const char* buffer, size_t length) {
   
   auto op_data = construct_persistent_object<FlashOpData>(context);
   auto parser = CustomOptionParser(buffer, length);
-  op_data->address = parser.parseNamedCustomOption("address").AsInt32();
-  op_data->bytes   = parser.parseNamedCustomOption("bytes").AsInt32();
+  op_data->addr = parser.parseNamedCustomOption("addr").AsInt32();
+  op_data->size   = parser.parseNamedCustomOption("size").AsInt32();
   tflite::micro::xcore::XCoreInterpreter* xint = reinterpret_cast<tflite::micro::xcore::XCoreInterpreter*>(context->impl_);
   op_data->c_flash = xint->c_flash;
   op_data->name    = "XC_Load_Flash_v2";
```

Comment:
why v2?

Reply:
That's a very good point. v2 should be removed.